hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/PersistenceTest.java

```diff
@@ -2380,4 +2379,108 @@ public void test0140EventTags() throws Exception {
         assertEquals("EventTag2Value", event.getTags().get("EventTag2Name"));
     }
 
+    // These tests would be nice in a separate clas but I couldn't figure how to get multiple test classes
+    // to run embedded Cassandra without stepping on each other.
```

Comment:
Note that to run these tests -Dtest-env=perf is needed.
This is automatically ran by travis CI.